app/browser/api/ledger.js

```diff
@@ -1103,14 +1107,18 @@ const backupKeys = (state, backupAction) => {
     includeAllFiles: false
   }, (files) => {
     const file = files && files.length ? files[0] : null
+    let success = false
     if (file) {
       try {
         fs.writeFileSync(file, message)
+        success = true
+        appActions.onLedgerBackupSuccess(success)
```

Comment:
I think that you can do just `appActions.onLedgerBackupSuccess(true)` and remove `success` variable, because `false` is not used